app/_locales/es/messages.json

```diff
@@ -929,5 +929,579 @@
+    "message": "Hay que usar MetaMask en Google Chrome para poder conectarse con tu Monedero Físico."
+  },
+  "confirmationTime": {
+    "message": "Tiempo de confirmación (seg)"
+  },
+  "connectHardwareWallet": {
+    "message": "Conectar Monedero Físico"
+  },
+  "connect": {
+    "message": "Conectar"
+  },
+  "connectRequest": {
+    "message": "Petición para Conectar"
+  },
+  "connecting": {
+    "message": "Conectándose..."
+  },
+  "connectingTo": {
+    "message": "Conectánodse a $1"
+  },
+  "connectToLedger": {
+    "message": "Conectarse al Ledger"
+  },
+  "connectToTrezor": {
+    "message": "Conectarse al Trezor"
+  },
+  "copyAddress": {
+    "message": "Copiar la dirección al portapapeles"
+  },
+  "currentLanguage": {
+    "message": "Idioma Actual"
+  },
+  "customGasSubTitle": {
+    "message": "Aumentar la comisión puede hacer que el tiempo de procesamiento se disminuya, pero no está garantizado."
+  },
+  "customToken": {
+    "message": "Token Personalizado"
+  },
+  "downloadGoogleChrome": {
+    "message": "Descargar Google Chrome"
+  },
+  "dontHaveAHardwareWallet": {
+    "message": "¿No tienes un monedero físico?"
+  },
+  "ensNameNotFound": {
+    "message": "No se localizó el nombre ENS"
+  },
+  "enterPasswordContinue": {
+    "message": "Intoducir contraseña para seguir"
+  },
+  "eth": {
+    "message": "ETH"
+  },
+  "estimatedProcessingTimes": {
+    "message": "Tiempo Previsto de Procesamiento"
+  },
+  "expandView": {
+    "message": "Ampliar Vista"
+  },
+  "fast": {
+    "message": "Rápido"
+  },
+  "faster": {
+    "message": "Más Rápido"
+  },
+  "fastest": {
+    "message": "Lo Mas Rápido"
+  },
+  "feeChartTitle": {
+    "message": "Previsiones en Vivo de las Comisiones por Transacciones"
+  },
+  "forgetDevice": {
+    "message": "Olvidar a este dispositivo"
+  },
+  "functionType": {
+    "message": "Tipo de Función"
+  },
+  "gasUsed": {
+    "message": "Gas Usado"
+  },
+  "gasPriceExtremelyLow": {
+    "message": "Precio de Gas Excesivamente Bajo"
+  },
+  "gasPriceNoDenom": {
+    "message": "Precio de Gas"
+  },
+  "generatingTransaction": {
+    "message": "Generando la Transacción"
+  },
+  "getHelp": {
+    "message": "Pedir Ayuda."
+  },
+  "hardware": {
+    "message": "hardware"
+  },
+  "hardwareWalletConnected": {
+    "message": "Se ha conectado el monedero físico"
+  },
+  "hardwareWallets": {
+    "message": "Conectarle un monedero físico"
+  },
+  "hardwareWalletsMsg": {
+    "message": "Seleccionar un monedero físico que quieres usar con MetaMask"
+  },
+  "havingTroubleConnecting": {
+    "message": "¿Tienes problemas para hacer la conexión?"
+  },
+  "hexData": {
+    "message": "Datos de formato Hex"
+  },
+  "history": {
+    "message": "Historial"
+  },
+  "importUsingSeed": {
+    "message": "Importar usando la frase semilla de la cuenta"
+  },
+  "initialTransactionConfirmed": {
+    "message": "La red confirmó tu transacción inicial. Hazle clic en OK para volver."
```

Comment:
"La red confirmó tu transacción inicial. Haz click en OK para volver."

Reply:
Hahaha, I'm not going to get into a leista debate :-) I'm fine with 'haz clic', it's probably more universal